ARM-wvd-templates/HCI/CreateHciHostpoolTemplate.json

```diff
@@ -0,0 +1,490 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+      "hostPoolName": {
+        "type": "string",
+        "metadata": {
+          "description": "The name of the Hostpool to be created."
+        }
+      },
+      "hostpoolFriendlyName": {
+        "type": "string",
+        "metadata": {
+          "description": "The friendly name of the Hostpool to be created."
+        },
+        "defaultValue": ""
+      },
+      "hostpoolDescription": {
+        "type": "string",
+        "metadata": {
+          "description": "The description of the Hostpool to be created."
+        },
+        "defaultValue": ""
+      },
+      "location": {
+        "type": "string",
+        "metadata": {
+          "description": "The location where the resources will be deployed."
```

Comment:
Change to:
Enter a region where Hostpool, workspace, virtual machines will be created. The metadata for these objects will be stored in the geography associated with the region. Supported regions: eastus and westeurope.